dataconf/utils.py

```diff
@@ -169,7 +172,11 @@ def __parse(value: any, clazz, path, strict, ignore_unexpected):
     if clazz is str:
         return __parse_type(value, clazz, path, isinstance(value, str))
 
-    if issubclass(clazz, Enum) or issubclass(clazz, IntEnum):
+    if clazz is Any:
+        # At this point, abandon type enforcement
+        return value
```

Comment:
@slyons Last comment before we get that merged: we discussed in the [past](https://github.com/zifeo/dataconf/issues/34) whether this library should support `Any`. I believe the best behaviour would be to disallow `Any` and throw a dedicated exception whenever this type is encountered. If we allow it, the user will anyway get a `ConfigTree` type as result (he could request explicitly that Type), not really `Any`. Do you see any scenario where this would become an issue?

Reply:
@zifeo I didn't see a version upgrade from the last PR. Will it be released or just a main change and release coming later?

Reply:
@dwsmith1983 We can batch multiple features: as soon as we get this PR merge, we can [trigger](https://github.com/zifeo/dataconf/actions/workflows/release-request.yaml) a new release.

Reply:
@slyons Having a flag any_as_config_tree could work for me 👍. I am still not convainced there is a difference between a and b in the following:
'''python
@dataclass
class Same:
  a: ConfigTree # will get a ConfigTree class
  b: Any # will get the same type
'''
However if you both feel this would work behind a flag, let's go for it. I would nonetheless put this in another PR if possible.